gourmet/backends/db.py

```diff
@@ -894,28 +894,33 @@ def get_unique_values (self, colname,table=None,**criteria):
                   ]
         return [x for x in retval if x is not None] # Don't return null values
 
-    def get_ingkeys_with_count (self, search={}):
+    def get_ingkeys_with_count (self, search=None):
         """Get unique list of ingredient keys and counts for number of times they appear in the database.
         """
+        if search is None:  # replaced mutable default argument: search={}
```

Comment:
I think this comment is not so helpful beyond this PR

Reply:
I should have added the comment to the PR. I will remove the comment from the code.